controllers/controllers/workloads/env/builder_test.go

```diff
@@ -350,6 +350,10 @@ var _ = Describe("EnvBuilder", func() {
 						"Name":  Equal("PORT"),
 						"Value": Equal("1234"),
 					}),
+					MatchFields(IgnoreExtras, Fields{
+						"Name":  Equal("CF_INSTANCE_PORTS"),
+						"Value": ContainSubstring("\"internal\":1234"),
```

Comment:
How about we use [`MatchJSON`](https://onsi.github.io/gomega/#matchjsonjson-interface) instead of `ContainSubstring`? thus we could ensure that the content of the value is a valid json

Reply:
Done! I suppose we could also create a struct and json.Marshall it... but i'm note sure if that's better as json shouldn't change?